test/assets/app-ingress/namespace.yaml

```diff
@@ -2,3 +2,5 @@ kind: Namespace
 apiVersion: v1
 metadata:
   name: cluster-ingress-test
+  labels:
+    openshift.io/cluster-ingress-operator-test: ""
```

Comment:
Reserved for future use: should make it easier to identify test namespaces for cleanup.